app/src/main/java/com/dessalines/rankmyfavs/db/AppDB.kt

```diff
@@ -12,7 +12,9 @@ import java.util.concurrent.Executors
 
 @Database(
     version = 2,
-    entities = [AppSettings::class, FavList::class, FavListItem::class, FavListMatch::class],
+    entities = [
```

Comment:
This adds new tables, and is a DB change, so you need to up the version number. I also don't see any migration adding the table and its columns.